static/app/views/alerts/rules/metric/details/sidebar.tsx

```diff
@@ -205,6 +221,13 @@ export function MetricDetailsSidebar({
             keyName={t('Environment')}
             value={<OverflowTableValue>{rule.environment ?? '-'}</OverflowTableValue>}
           />
+          {rule.monitorType === MonitorType.ACTIVATED &&
+            typeof rule.activationCondition !== 'undefined' && (
```

Comment:
nit: i believe you can just check to see if it's undefined (i'm guessing we can't just do !rule.activationCondition cause there's a `0` case?)
```suggestion
            rule?.activationCondition !== undefined && (
```

Reply:
yea i think either works